src/tests/issues.targets

```diff
@@ -1172,9 +1172,6 @@
         <ExcludeList Include="$(XunitTestBinBase)/Interop/PInvoke/Variant/VariantTest/*">
             <Issue>https://github.com/dotnet/runtimelab/issues/155: VARIANT marshalling</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/Interop/StringMarshalling/AnsiBSTR/AnsiBStrTest/*">
```

Comment:
Did you run this test locally and verified that it is passing?

Reply:
Yes. I do verify now.